server/app/assets/javascripts/file_upload.ts

```diff
@@ -0,0 +1,37 @@
+window.addEventListener('load', (event) => {
```

Comment:
Do we have test coverage for this?

Reply:
There isn't currently any unit test coverage of our TS files :(. However, there is browser test coverage of the "requiredness" constraints that file is responsible for here: https://github.com/seattle-uat/civiform/blob/1c9ed6afa33336531fdf0eeca0729717007415fa/browser-test/src/file.test.ts#L75-L84